StripeFinancialConnections/StripeFinancialConnections/Source/Native/AccountPicker/RetrieveAccountsLoadingView.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  RetrieveAccountsLoadingView.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 1/8/24.
+//
+
+import Foundation
+import UIKit
+
+final class RetrieveAccountsLoadingView: UIView {
+
+    init(institutionIconUrl: String?) {
+        super.init(frame: .zero)
+        let paneLayoutView = PaneLayoutView(
+            contentView: PaneLayoutView.createContentView(
+                iconView: {
+                    if let institutionIconUrl = institutionIconUrl {
+                        let institutionIconView = InstitutionIconView()
+                        institutionIconView.setImageUrl(institutionIconUrl)
+                        return institutionIconView
+                    } else {
+                        return nil
+                    }
+                }(),
+                title: STPLocalizedString(
+                    "Retrieving accounts...",
+                    "The title of the loading screen that appears when a user just logged into their bank account, and now is waiting for their bank accounts to load. Once the bank accounts are loaded, user will be able to pick the bank account they want to to use for things like payments."
+                ),
+                subtitle: nil,
+                contentView: {
+                    let verticalStackView = UIStackView(
+                        arrangedSubviews: [
+                            ShimmeringAccountPickerRow(),
```

Comment:
Is this expected? if ShimmeringAccountPickerRow is only used here can we just pack the 4 dummy rows within it?

Reply:
Could you expand on what you mean by this? 

The `ShimmeringAccountPickerRow` is created to add some extra separation of code. It's marked as `private` so it can't be referenced outside of this file. Then here we just put 4 rows in a Vertical Stack to make a list of them.

Reply:
I meant if there's always 4 of them added to `verticalStackView` and it's private, can we instead create a single view with the 4 rows hardcoded within

Reply:
Sorry, you'd maybe have to type out the idea 😅 

I could see someone creating an extra view to encapsulate the logic, but at the moment I don't see easy ways to simplify it. The `UIStackView` is there for layout. `ShimmeringAccountPickerRow` is to put the "4 row code" to be reusable 4 times (also utilizes `ShimmeringView` subclass which likely will be used elsewhere).

```
// just quick psuedocode
final class ShimmeringAccountPickerList: UIView {

    init() {
         super.init(frame: .zero)
                    let verticalStackView = UIStackView(
                        arrangedSubviews: [
                            ShimmeringAccountPickerRow(),
                            ShimmeringAccountPickerRow(),
                            ShimmeringAccountPickerRow(),
                            ShimmeringAccountPickerRow(),
                        ]
                    )
                    verticalStackView.axis = .vertical
                    verticalStackView.spacing = 16
                   addSubview(verticalStackView)
     }
}
```

The `ShimmeringView` will likely eventually be moved out to a separate file as there are other parts that will need some "shimmering," but I kept it here until that happens.

Reply:
if it will be used elsewhere, then it makes sense to do it this way